examples/quickstart_mxnet/requirements.txt

```diff
@@ -1,3 +1,2 @@
-flwr~=1.4.0
-mxnet~=1.9.1
-numpy~=1.21.1
+flwr==0.17.0
```

Comment:
I just merged `main` and resolved a conflict, and now I am confused. Is this correct @danielnugraha ?